samples/cellular/nrf_cloud_multi_service/overlay-nrf7002ek-wifi-scan-only.conf

```diff
@@ -9,7 +9,7 @@
 # Enable the Wi-Fi location method:
 CONFIG_LOCATION_TRACKING_WIFI=y
 CONFIG_LOCATION_METHOD_WIFI=y
-# Align this with CONFIG_NRF700X_SCAN_LIMIT.
+# Align this with CONFIG_WIFI_MGMT_SCAN_MAX_BSS_CNT.
```

Comment:
I wonder if the location lib could be updated to reference this directly?

Reply:
Originally I didn't refer to CONFIG_NRF700X_SCAN_LIMIT directly but had this own Kconfig because we still had to support ESP WiFi chip.